### PR TITLE
Use the dev version of enc package to fix handling non-ASCII characters on Windows

### PR DESCRIPTION
**Pull request by yutannihilation**
related: #329

### CODE REVIEW

**Comment by lorenzwalthert** [comment]
Is this the behaviour of interest here? I think the source needs to be changed to 
```
latin_string <- "Gl\u00fcck + 1"
```

**Comment by yutannihilation** [comment]
Thanks for the hint! Sorry, I compared the result with the wrong string... But, the source should be "ugly" text that needs to be styled, otherwise the file stays unchanged.

**Comment by lorenzwalthert** [comment]
Let's wait until a new version of `enc` is on CRAN that includes https://github.com/krlmlr/enc/pull/18, rerun CI and merge afterwards.

**Comment by yutannihilation** [comment]
Thanks. I wonder if it's better to add `krlmlr/enc` on `Remotes` for now if the next release will not come soon. What do you say?

**Comment by yutannihilation** [comment]
I just added a tweak to skip this test on Windows as suggested on https://github.com/krlmlr/enc/pull/18#discussion_r163591997 (which I've forgot to reflect here).

**Comment by lorenzwalthert** [comment]
Yes, you are right. So people installing via `remotes` or `devtools` from GitHub can get the fix. Do you want to expand this PR in this regard?

**Comment by yutannihilation** [comment]
Sure, I'll include it here 👍

**Comment by yutannihilation** [comment]
Confirmed the tests now pass. I believe this is ready to merge now :)

File: tests/testthat/test-encoding.R [hunk 1]
```diff
@@ -0,0 +1,21 @@
+context("non-ASCII characters are handled properly")
+
+test_that("non-ASCII characters are handled properly", {
+  skip_if(.Platform$OS.type != "windows")
```
Comment (lorenzwalthert) [inline tests/testthat/test-encoding.R]: Is there a particular reason not to test on unix? If not, I think we can reduce complexity here and test on all platforms I think.

Comment (lorenzwalthert) [reply tests/testthat/test-encoding.R]: Ah ok, did not see https://github.com/krlmlr/enc/pull/18#discussion_r163591997

**Comment by lorenzwalthert** [comment]
Thanks @yutannihilation.

**Comment by yutannihilation** [comment]
Thanks!

**Pull request merged**